dunereco/DUNEWireCell/dune-vd-coldbox/params.jsonnet

```diff
@@ -124,22 +124,11 @@ base {
     //                    // theoretical elec resp (14mV/fC): 36.6475 ADC*tick/1ke
     //   shaping: 2.2 * wc.us,
     // },
-    // elec: super.elec {
-    //      type: "JsonElecResponse",
-    //      filename: "dunevd-coldbox-elecresp-top-psnorm_400.json.bz2",
-    //      postgain: 1.0,
-    // },
-    elec: if std.extVar('active_cru')=='tde'
-          then super.elec {
-              type: "JsonElecResponse",
-              filename: "dunevd-coldbox-elecresp-top-psnorm_400.json.bz2",
-              postgain: 1.0,
-          }
-          else super.elec {
-              postgain: 1.1365, // pulser calibration: 41.649 ADC*tick/1ke
-                               // theoretical elec resp (14mV/fC): 36.6475 ADC*tick/1ke
-              shaping: 2.2 * wc.us,
-          },
+    elec: super.elec {
+         type: "JsonElecResponse",
+         filename: "cold-elec-response-gain14-shape2d2.json.bz2",
+         postgain: 1.0,
+    },
```

Comment:
We recently added an extVar "active_cru", could you keep it up to date?

Reply:
Sure, @wenqiang-gu, I will revert this. This was copied from here:
https://github.com/WireCell/wire-cell-toolkit/blob/564fc1c4be8c8abe345a13251062a9c0a1fdd191/cfg/pgrapher/experiment/dune-vd-coldbox/params.jsonnet
Which means, the `wire-cell-toolkit` needs some updates.